urllib3/exceptions.py

```diff
@@ -180,6 +180,14 @@ class SNIMissingWarning(HTTPWarning):
     pass
 
 
+class DependencyWarning(HTTPWarning):
```

Comment:
I'm inclined to leave it: if you want to filter out all urllib3 warnings, I feel like that should be do-able just as before.